src/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "WorldBrain's Memex",
     "short_name": "Memex",
-    "version": "0.1.0",
+    "version": "0.1.1",
```

Comment:
Was this accidentally staged in this commit? It's been annoying me and have accidentally committed it multiple times as well. You should be able to fix it then do a simple `git commit --amend`, as its from your latest commit (or just do a standard `git rebase -i` and mark the commit as `edit`/`e`)

Reply:
Ahhh I commited it by mistake. Sorry! 
I will amend it right now.